core/optic/shared/src/main/scala/com/useoptic/ux/DiffPreviewer.scala

```diff
@@ -28,6 +28,8 @@ object DiffPreviewer {
   ///@todo make return optional
   def previewDiff(jsonLike: Option[JsonLike], spec: RfcState, shapeIdOption: Option[ShapeId], diffs: Set[ShapeDiffResult]): Option[SideBySideRenderHelper] = shapeIdOption map { shapeId =>
 
+    implicit val Resolvers = new MemoizedResolvers(spec)
```

Comment:
once I play around with this a little bit I'll probably change this to be a parameter since it should be constructed once when we build the `RfcContext` alongside the `cachedQueryResults`